scripts/install.sh

```diff
@@ -18,8 +23,14 @@ oc apply -f ./deploy/crds/ApplicationMonitoring.yaml
 
 # Cluster Roles & RoleBindings
 oc apply -f ./deploy/roles
-oc apply -f ./deploy/operator_roles/
+oc apply -f ./deploy/operator_roles/service_account.yaml
+oc apply -f ./deploy/operator_roles/role.yaml
+oc apply -f ./deploy/operator_roles/role_binding.yaml
 
 # AMO Deployment
+until oc auth can-i create prometheus -n application-monitoring --as system:serviceaccount:application-monitoring:application-monitoring-operator; do
```

Comment:
This may not be as necessary now with the rest of the fixes added here. That said, maybe this entire file isn't necessary, and could just be inlined in the Makefile. I'm happy to remove this and/or inline the rest of the script in the Makefile.